### PR TITLE
Minutes 2024-09-12

### PR DESCRIPTION
**Pull request by DaniMori**

### CODE REVIEW

**Comment by kevolve** [comment]
Looks good!

File: doc/meetings/2024-09-12_Minutes.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+-   Find out about split rooms [ASAP]
+
+-   Add technical assistance info to slides [10-02]
+
+-   Update contents [09-20]
+
+## KBN
+
+-   Propose system for providing technical assistance [09-30]
```
Comment (kevolve) [inline doc/meetings/2024-09-12_Minutes.md:84]: Added to content-updates branch!

File: doc/meetings/2024-09-12_Minutes.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+-   Send DMC UQ logo [ASAP]
```
Comment (kevolve) [inline doc/meetings/2024-09-12_Minutes.md:86]: Done!

File: doc/meetings/2024-09-12_Minutes.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+-   Test contents [09-30]
```
Comment (kevolve) [inline doc/meetings/2024-09-12_Minutes.md:88]: Done! Modified R code a little and one slide was going off the page (R environment) so divided into two

**Review by kevolve — APPROVED** [review]
Looks good!

File: doc/meetings/2024-09-12_Minutes.md (hunk 3 above)
Comment (DaniMori) [reply doc/meetings/2024-09-12_Minutes.md:88]: Cool! I'll have a look in that PR ;)

File: doc/meetings/2024-09-12_Minutes.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+**Schedule:** 09:00 UTC - 10:00 UTC
+
+# Action points
+
+## Decide workshop approach
+
+### Discussion
+
+-   Proposed instructor roles: DMC facilitator, JBN co-facilitator (moderation,
```
Comment (DaniMori) [inline doc/meetings/2024-09-12_Minutes.md:25]:
```suggestion
-   Proposed instructor roles: DMC facilitator, KBN co-facilitator (moderation,
```

**Comment by DaniMori** [comment]
@kevolve I had you misspelled as "JBN" instead of "KBN", but corrected it myself. I assume there's no need to re-approve, so I'll just merge the PR 😉

**Pull request merged**